backend/src/agent/persistence/main.go

```diff
@@ -125,4 +127,5 @@ func init() {
 		"/apis/v1beta1", "The base path for the ML pipeline API server.")
 	flag.StringVar(&namespace, namespaceFlagName, "", "The namespace name used for Kubernetes informers to obtain the listers.")
 	flag.Int64Var(&ttlSecondsAfterWorkflowFinish, ttlSecondsAfterWorkflowFinishFlagName, 604800 /* 7 days */, "The TTL for Argo workflow to persist after workflow finish.")
+	flag.IntVar(&numWorker, numWorkerName, 2, "Number of worker for sync job.")
```

Comment:
Would you mind adding an explict usage of this new argument in https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml as both an example and a default value.

Reply:
Thanks!